atomwrappers.py

```diff
@@ -75,6 +78,7 @@ class CListLink(CLink):
 
 class CGeneNode(CNode):
     atom_type = 'GeneNode'
+    name = get_current_symbol(CNode)
```

Comment:
It looks like a bug: get_current_symbol expects a string. Besides what is the point of setting name here?

Reply:
sorry it is a bug